custom/test/PassageTest.php

```diff
@@ -99,6 +99,24 @@ public function testGetUser()
         $this->assertEquals($user['id'], $this->userId);
     }
 
+    public function testGetUser()
+    {
+        $email = 'test-create@passage.id'
+        $userRequest = new CreateUserRequest(array(
+            'email' => $email,
+        ));
+
+        $createUser = $this->passageClient->createUser($userRequest);
+        
+        $user = $this->passageClient->getUser($createUser['id']);
+        $this->assertEquals($user['id'],$createUser['id']);
+
+        $userByIdentifier = $this->passageClient->getUserByIdentifier($email);
+        $this->assertEquals($userByIdentifier['id'],$createUser['id']);
+
+        $this->assertEquals($userByIdentifier, $user)
+    }
+
```

Comment:
Can we also test the correct error is thrown if no user is found. I should have said that for node too.